docs/reference/modules/discovery/discovery-settings.asciidoc

```diff
@@ -3,6 +3,13 @@
 
 Discovery and cluster formation are affected by the following settings:
 
+`cluster.initial_master_nodes`::
```

Comment:
I find it harder to scan a list like this when there's no obvious sorting, but since I know there are larger discussions around reorganizing settings reference pages it's likely not a big deal.